app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt

```diff
@@ -116,9 +118,9 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
     }
 
     var onNavigationEvent: ((
-        event: NavigationEvent,
-        value: String?,
-        autocompleteResult: InlineAutocompleteEditText.AutocompleteResult?
+            event: NavigationEvent,
```

Comment:
Is this indentation correct?

Reply:
I think so? 8 line indentation as per ktlint suggestion